pkgs/top-level/all-packages.nix

```diff
@@ -5616,6 +5616,8 @@ with pkgs;
 
   tpm-luks = callPackage ../tools/security/tpm-luks { };
 
+  trezorCipherKeyValue = callPackage ../tools/misc/trezorCipherKeyValue { };
```

Comment:
these shouldn't be camelCased, see the nixpkgs manual¹²

> Names of files and directories should be in lowercase, with dashes between words — not in camel case. For instance, it should be all-packages.nix, not allPackages.nix or AllPackages.nix.

¹https://nixos.org/nixpkgs/manual/#sec-package-naming
²https://nixos.org/nixpkgs/manual/#sec-organisation

Reply:
This one I can try to do, but I spent 2 days to get it work. I will try do just rename if that will work, then ok